plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/PullParams.java

```diff
@@ -43,7 +43,7 @@ public PullParams withTag(String tag) {
 
     /**
      * @param registry
-     *         registry url
+     *         host and port of registry, e.g. localhost:5000
```

Comment:
Please mention that it is optional in case of official docker registry - hu.docker.com